test_resolwe_runtime_utils.py

```diff
@@ -15,9 +15,10 @@
 # pylint: disable=missing-docstring
 import json
 from unittest import TestCase
+from unittest.mock import patch
```

Comment:
The `unittest` module in Python2 does not include `mock`. Should we support Python2? Which Python do we use in docker?